### ISSUE TITLE
Geppetto instances getGeometryType

### ISSUE THREAD

**Issue by ddelpiano** [openworm/geppetto-client#317]
on the same line of the setGeometryType made available from the SceneController, we should have a getter method that allow us to get the geometry type (lines, cylinders, etc) for a specific instance.

@tarelli my understanding at the moment is that this is possible only going through the rendered scene in the 3dengine and once we found the instance check the geometry used. I thought of storying the geometryType for a specific instance when this is added based on the canvas parameters to render new instances and store this in the instance, and once the setter method gets called store the change in the instances itself to make the getter lighter. Any cons in this that you might think about? Thx.

**Comment by tarelli** [comment]
@ddelpiano yes there is no such a getter but also, we are rewriting the canvas :) how urgent is this?

**Comment by ddelpiano** [comment]
not urgent at all, we discussed this in yesterday's sprint call and we decided to keep track of this, at the moment we have a workaround within vfb but it would be better to replace that with such getter whenever this will be available. Thx!
